kustomize/overlays/ci/shard/event-bridge-shard-operator-secrets.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: addon-smart-events-operator-parameters
+type: Opaque
+stringData:
+  EVENT_BRIDGE_SHARD_ID: 150b23e9-0b34-45b7-a91a-5ec388d03a1d
```

Comment:
Can you please use `toReplace` value also here?

Reply:
Also please adjust [Manager-CI.yaml](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/1089/files#diff-0facce3621ef7b3815a47809a968cb4d1dafc823dbaea4f1ab136788e6dea998) to replace this place

Reply:
done